setup.py

```diff
@@ -68,37 +68,26 @@ def dmf_data_files(root: str = DMF_DATA_ROOT) -> List[Tuple[str, List[str]]]:
     # Concrete dependencies go in requirements[-dev].txt
     install_requires=[
         # idaes core / dmf
-        "backports.shutil_get_terminal_size",
-        "bunch",
         "click>=8",
         "colorama",
-        "distro",  # help identify linux distros for binary downloads
-        "flask",  # for ui/fsvis
-        "flask-cors",
         "jupyter",
         "lxml",
         "matplotlib",
         "nbconvert",
         "nbformat",
         "numpy",
-        "networkx",
```

Comment:
I don;t think we should necessarily have this as a dependency, but this is something we might rely on implicitly - it gets used in the Pyomo SequentialDecomposition tool (but the dependency probably should be handled there).